client/components/redux-forms/redux-form-text-input/index.jsx

```diff
@@ -8,21 +8,25 @@ import { omit } from 'lodash';
 /**
  * Internal dependencies
  */
+import FormInputValidation from 'components/forms/form-input-validation';
 import FormTextInput from 'components/forms/form-text-input';
 
 class ReduxFormTextInput extends Component {
 	static propTypes = {
 		name: PropTypes.string,
 	};
 
-	renderTextInput = ( { input: { onChange, value } } ) => {
+	renderTextInput = ( { input: { onChange, value }, meta: { touched, error } } ) => {
 		const otherProps = omit( this.props, 'name' );
 
 		return (
-			<FormTextInput
-				{ ...otherProps }
-				onChange={ this.updateTextInput( onChange ) }
-				value={ value } />
+			<span>
+				<FormTextInput
+					{ ...otherProps }
+					onChange={ this.updateTextInput( onChange ) }
+					value={ value } />
+				{ touched && error && <FormInputValidation isError text={ error } /> }
```

Comment:
The `FormTextInput` could also use a `isError={ !! ( touched && error ) }` property. It will then display with a nice red border.